vochain/transaction/proof.go

```diff
@@ -115,21 +115,21 @@ func VerifyProofOffChainTree(process *models.Process, proof *models.Proof,
 		if len(leafKey) > censustree.DefaultMaxKeyLen {
 			leafKey = leafKey[:censustree.DefaultMaxKeyLen]
 		}
-		valid, err := tree.VerifyProof(hashFunc, leafKey, p.LeafWeight, p.Siblings, censusRoot)
+		valid, err := tree.VerifyProof(hashFunc, leafKey, p.AvailableWeight, p.Siblings, censusRoot)
 		if !valid || err != nil {
 			return false, nil, err
 		}
 		// Legacy: support p.LeafWeight == nil, assume then value=1
-		if p.LeafWeight == nil {
+		if p.AvailableWeight == nil {
 			return true, bigOne, nil
 		}
 
-		factoryWeight := arbo.BytesToBigInt(p.LeafWeight)
-		if p.VotingWeight == nil {
+		factoryWeight := arbo.BytesToBigInt(p.AvailableWeight)
```

Comment:
Maybe it is a good time to change factoryWeight by the new naming

Reply:
done, i will check the rest of the code to ensure that we use the same naming